spec/busted-ci-helper.lua

```diff
@@ -1,10 +1,16 @@
 -- busted-ci-helper.lua
-
-local busted = require 'busted'
-local cjson = require 'cjson'
-local socket_unix = require 'socket.unix'
+local busted = require("busted")
+busted.subscribe({ "suite", "start" }, function()
+  require("spec.helpers").cleanup_kong()
```

Comment:
```suggestion
  require("spec.helpers").cleanup_kong()
  return nil, true --continue
```
I think we should return `nil, true` to continue execution of any other subscribers of that event, otherwise this will stop that event's propagation.

Reply:
Thanks!